### PR TITLE
Do not enforce Yajl and try to use MultiJson

### PR DESCRIPTION
**Pull request by pschambacher**
:octopus::computer:

/cc @grosser 

### Steps to merge
* [ ] :+1: from the team

### Risks
* Low: error parsing or dumping ticket sharing

### CODE REVIEW

**Comment by grosser** [comment]
the gemfiles seem like a bit of overkill ... how about switching jsons in 1 test and then testing that the output is the same ?

**Comment by pschambacher** [comment]
Can do

**Comment by pschambacher** [comment]
@grosser Just removed the commit with multiple gemfiles. Not necessary I think.

File: lib/ticket_sharing/json_support.rb [hunk 1]
```diff
@@ -1,14 +1,20 @@
-require 'yajl'
-
 module TicketSharing
+  begin
```
Comment (grosser) [inline lib/ticket_sharing/json_support.rb]: prefer `Gem.available?('multi_json')` instead of causing a raise + rescue

Comment (grosser) [inline lib/ticket_sharing/json_support.rb]: could also turn this into an assignment like `JSON_PARSER = if xxx`

**Comment by grosser** [comment]
👍

**Comment by pschambacher** [comment]
`Gem.available?` was removed. Recommended behavior is to rescue...

**Pull request merged**